src/main/drivers/rangefinder/rangefinder_lidartf.h

```diff
@@ -26,3 +26,4 @@
 
 bool lidarTFminiDetect(rangefinderDev_t *dev);
 bool lidarTF02Detect(rangefinderDev_t *dev);
+bool lidarTFNovaDetect(rangefinderDev_t *dev);
```

Comment:
With reasonable refactoring (array of device parameters + rangefinderHardwareToUse in it), this can be replaced with single function.